src/js/intlTelInput.js.ejs

```diff
@@ -246,6 +250,11 @@ Plugin.prototype = {
       // this is useful in lots of places
       this.countryListItems = this.countryList.children(".country");
     }
+    
+    // add the RTL class if requested
+    if(this.options.rtl === true) {
+    	this.telInputRootElement.addClass("iti-rtl");
```

Comment:
It's only used once and is the direct parent now.
Later that may change and it could potentially break the RTL styling by placing the class on the wrong element.
Also, it's a good idea to keep reference to the top-most element of the component and introduces no performance issue.
I'd keep this as is.